src/quo2/components/drawers/documentation_drawers/style.cljs

```diff
@@ -0,0 +1,10 @@
+(ns quo2.components.drawers.documentation-drawers.style)
+
```

Comment:
hmm, this is lacking the color variations? There should be light, dark and shell themes. 🤔 or you'll do that in a follow up?

Reply:
I will add it.

Reply:
I'm using `text`, `button` and `bottom-sheet` components so dark & light variations will be handled by those components. For `shell` variant I have updated the bottom-sheet with a `blur?` option. Please check the the `bottom-sheet/view` file. Design only has shell variant in the dark theme so I think it will only be used in dark theme. Should we confirm this with the design team?

Reply:
Yes, that makes sense about the components.
shell theme is another variation of dark theme, let's confirm with them just to make sure it's all setup as expected 👍 .